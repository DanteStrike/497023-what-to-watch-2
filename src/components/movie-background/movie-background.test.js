import React from "react";
import renderer from "react-test-renderer";
import MovieBackground from "./movie-background.jsx";


it(`Render correctly MovieBackground component`, () => {
  const component = renderer
    .create(
        <MovieBackground/>
    ).toJSON();

  expect(component).toMatchSnapshot();
});
