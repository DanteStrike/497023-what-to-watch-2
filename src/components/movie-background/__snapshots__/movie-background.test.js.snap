// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly MovieBackground component 1`] = `
<div
  className="movie-card__bg"
>
  <img
    alt="any"
    src="url"
    style={
      Object {
        "backgroundColor": "color",
      }
    }
  />
</div>
`;
